README.md

```diff
@@ -72,9 +72,23 @@ If you absolutely have to share it with a fellow contributor, who for some reaso
 
 If you have mistakenly done so, you might want to remove and purge [such sensitive data](https://help.github.com/articles/remove-sensitive-data/).
 
-### Contributions
+### Contributions & Roadmap
 
-For more info on how to contribute, check out contribution guide.
+We have developed an initial set of meta challenges required for Python in this [Google Doc](https://docs.google.com/document/d/1lNi2bKms52uMtvwxn3znnspz08UUwuSTMYXKDyiUd_k/edit). This note will serve as a roadmap of how we will proceed and contribute in this repo, not only for Python but for all backend languages.
+
+#### Deliverables
+
+The final UI framework, using [REPL.it](https://repl.it), will send to the repl server two sets of code concatenated, one the user snippet, and another the test code, and then would check for all tests passed in the returned answer. So, we need to deliver two things per challenge :
+
+* The barebones code file, without solution
+* The unit test file, testing the above
+
```

Comment:
Maybe add a future deliverable here - that we need to create Wiki articles with solutions for these.